requirements.txt

```diff
@@ -1,5 +1,5 @@
 av==10.*
-ctranslate2>=3.17,<4
+ctranslate2>=3.21,<4
 huggingface_hub>=0.13
 tokenizers>=0.13,<0.15
```

Comment:
Add support to tokenizer 0.15 version, otherwise it is inconvenient to install faster-whisper and transformers/tokenizers

Reply:
do you know if any compatibility fixes are necessary? or will just `tokenizers>=0.13` suffice?

Reply:
I guess Guillaume did that because ctranslate2 relies heavily on the tokenizers, to prevent a new tokenizers version to introduce breaking change into faster-whisper. We can go for tokenizers>=0.13,<0.16

Reply:
We made some tests on ourside, the tokenizers 0.15 works normally, so: tokenizers>=0.13,<0.16 looks good for me.